authorization/role/service/role_management_service_blackbox_test.go

```diff
@@ -483,6 +483,112 @@ func (s *roleManagementServiceBlackboxTest) TestRevokeResourceRolesOK() {
 	assert.Len(s.T(), idRoles, 10)
 }
 
+func (s *roleManagementServiceBlackboxTest) TestPrivilegeCacheNotified() {
```

Comment:
should you also verify that another member of the same team does not have her cache marked as stale when someone else joins/leaves the team ?

Reply:
That's a good idea, I'll update the test to do that.

Reply:
thanks!